scripts/cli/tasks/closeMilestone.ts

```diff
@@ -0,0 +1,87 @@
+import axios from 'axios';
+import { Task, TaskRunner } from './task';
+
+interface CloseMilestoneOptions {
+  milestone: string;
+}
+
+const closeMilestoneTaskRunner: TaskRunner<CloseMilestoneOptions> = async ({ milestone }) => {
+  const gitHubUsername = process.env.GITHUB_USERNAME;
+  const gitHubToken = process.env.GITHUB_ACCESS_TOKEN;
+
+  if (!gitHubUsername || !gitHubToken) {
+    console.log('the close milestone operation needs a GITHUB_USERNAME and GITHUB_ACCESS_TOKEN environment variables');
+    return;
+  }
+
+  const client = axios.create({
+    baseURL: 'https://api.github.com/repos/grafana/grafana',
+    timeout: 10000,
+    auth: {
+      username: gitHubUsername,
+      password: gitHubToken,
+    },
+  });
+
+  const cherryPickLabel = 'cherry-pick needed';
+
+  if (!/^\d+$/.test(milestone)) {
+    console.log('Use milestone number not title, find number in milestone url');
+    return;
+  }
+
+  const milestoneRes = await client.get(`/milestones/${milestone}`, {});
+
+  const milestoneState = milestoneRes.data.state;
+
+  if (milestoneState === 'closed') {
+    console.log('milestone already closed. ✅');
+    return;
+  }
+
+  console.log('fetching issues/PRs of the milestone ⏬');
+
+  // Get all the issues/PRs with the label cherry-pick
+  // Every pull request is actually an issue
+  const issuesRes = await client.get('/issues', {
+    params: {
+      state: 'closed',
+      labels: cherryPickLabel,
+      per_page: 100,
+      milestone: milestone,
+    },
+  });
+
+  if (issuesRes.data.length < 1) {
+    console.log('no issues to remove label from');
+  } else {
+    console.log(`found ${issuesRes.data.length} issues to remove the cherry-pick label from 🔎`);
+  }
+
+  for (const issue of issuesRes.data) {
+    process.stdout.write(`🔧removing label from issue #${issue.number} 🗑...`);
+    const resDelete = await client.delete(`/issues/${issue.number}/labels/${cherryPickLabel}`, {});
+    if (resDelete.status === 200) {
+      process.stdout.write('done ✅\n');
+    } else {
+      console.log('failed ❌\n');
```

Comment:
Nitpick: Maybe continue with the `process.stdout.write`approach or maybe remove the `\n`? Or maybe you want 2 newlines :)
```suggestion
      console.log('failed ❌\n');
```

Reply:
Great shout